.vib/harbor-core/goss/vars.yaml

```diff
@@ -0,0 +1,13 @@
+binaries:
+  - harbor_core
+  - ini-file
+  - wait-for-port
+directories:
+  - paths:
```

Comment:
If I'm not mistaken, you are missing the checks for the following directories per the postunpack logic:
- ${HARBOR_CORE_VOLUME_DIR}/certificates
- ${HARBOR_CORE_VOLUME_DIR}/ca_download
- ${HARBOR_CORE_VOLUME_DIR}/psc

Reply:
Since we are performing a symlink, those directories stop to exist.

Reply:
My bad, you are already checking the symlinks in the `goss.yaml`. This comment doesn't apply then.